npm/private/npm_link_package_store.bzl

```diff
@@ -46,13 +58,25 @@ If set, takes precendance over the package name in the src npm_package_store.
     ),
 }
 
+_BIN_TMPL = """#!/bin/sh
+basedir=$(dirname "$(echo "$0" | sed -e 's,\\\\,/,g')")
+exec node "$basedir/{bin_path}" "$@"
+"""
+
```

Comment:
Unrelated to this PR, but do we need a windows `.cmd` equivalent for these like I did [here](https://github.com/angular/angular/pull/47135/files#diff-2bb203c749eb393e1596af31197ef437a63f87fd6ac87dc6222b89f70fee37cfR239-R243)?

Reply:
As a follow-up probably yes. For downstream work I'm going to use sh_test which works on Windows.